### PR TITLE
added link to collector helm charts

### LINKED ISSUE #993: Docs for Helm installation for Collector
**What needs to be changed?**
Provide documentation for Helm installation for Collector to Kubernetes.

**What is the name + path of the page that needs changed?**
N/A

**Additional context**
It currently only has docs on doing YAML deployments on https://opentelemetry.io/docs/collector/getting-started/#kubernetes

### PR DESCRIPTION
**Pull request by TylerHelmuth**
Fixes #993

### CODE REVIEW

**Comment by chalin** [comment]
Preview: https://deploy-preview-1243--opentelemetry.netlify.app/docs/collector/getting-started

File: content/en/docs/collector/getting-started.md [hunk 1]
```diff
@@ -90,7 +91,7 @@ $ kubectl apply -f https://raw.githubusercontent.com/open-telemetry/opentelemetr
 ```
 
 The example above is meant to serve as a starting point, to be extended and
-customized before actual production usage.
+customized before actual production usage. Use the [OpenTelemetry Helm Charts](https://github.com/open-telemetry/opentelemetry-helm-charts) for production-ready customization and installation.
```
Comment (chalin) [inline content/en/docs/collector/getting-started.md:94]: I'd suggest the following rewording:

> For production-ready customization and installation, see [OpenTelemetry Helm Charts](https://github.com/open-telemetry/opentelemetry-helm-charts).

**Review by chalin — COMMENTED** [review]
LGTM. See inline comment for a suggested copyedit.

@tomkerkhove, @open-telemetry/collector-approvers: WDYT?

File: content/en/docs/collector/getting-started.md (hunk 1 above)
Comment (TylerHelmuth) [reply content/en/docs/collector/getting-started.md]: Updated.

**Review by mx-psi — APPROVED** [review]

**Review by chalin — APPROVED** [review]
Thanks for the review and approval @mx-psi.

**Pull request merged**